rskj-core/src/test/java/co/rsk/peg/federation/NonStandardErpFederationsTest.java

```diff
@@ -1005,7 +997,7 @@ private void validateErpRedeemScript(
         assertEquals(ScriptOpCodes.getOpCode(String.valueOf(n)), script[index++]);
 
         assertEquals(ScriptOpCodes.OP_ENDIF, script[index++]);
-        assertEquals(Integer.valueOf(ScriptOpCodes.OP_CHECKMULTISIG).byteValue(), script[index++]);
+        assertEquals(Integer.valueOf(ScriptOpCodes.OP_CHECKMULTISIG).byteValue(), script[index]);
```

Comment:
Why didn't this change break the test since you didn't update anything else here?

Reply:
is using the same `index` value, just not incrementing it after

Reply:
Got it